tippicserver/playbooks/roles/tippic-server/templates/tippicserver/config.py.jinja2

```diff
@@ -26,6 +26,9 @@ P2P_TRANSFERS_ENABLED = {{ p2p_transfers_enabled }}
 P2P_MIN_KIN_AMOUNT = {{ p2p_min_kin_amount }}
 P2P_MAX_KIN_AMOUNT = {{ p2p_max_kin_amount }}
 
+DISCOVERY_APPS_ANDROID_URL = 'https://cdn.kinitapp.com/discovery_apps_android.json'
+DISCOVERY_APPS_OSX_URL = 'https://cdn.kinitapp.com/discovery_apps_osx.json'
```

Comment:
make this a parameter - so that it can be configured differently for stage / production